### DISCUSSION TITLE
Issue with BCL key

### DISCUSSION THREAD

**Discussion by melika-b** [BuildingComponentLibrary/bcl-manifest#26] [Q&A]
Hi,
Due to running the open studio, I copied BCL key you provided here:https://bcl.nrel.gov/documentation, but it doesn't work for me.
would you tell me what is the problem?
Thanks

**Comment by kflemin** [comment]
Hi @melika-b, were you able to resolve this issue? What OpenStudio application version do you have and what specific error are you getting? thanks!

**Comment by melika-b** [comment]
Hi Dear,

I wanted to upload "Reporting > openstudio result and Diagnostic". I
entered the BCL key you provided here:
https://bcl.nrel.gov/documentation#key (XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX), but
it doesn't work.
Would you guide me?
best regards,
Melika

**Reply by kflemin** [reply]
you may have better luck asking your question in the Unmet Hours forum: https://unmethours.com/questions/scope:all/sort:activity-desc/tags:bcl/page:1/

Also, sometimes the BCL goes down, have you tried again to make sure it still doesn't work? 
You could also try downloading the measure directly from the BCL and placing it in your local 'my measures' directory.

**Comment by melika-b** [comment]
By the way, the OpenStudio application version I use is 1.2.1

**Comment by melika-b** [comment]
Dear Sir or Madam,
Thank you for your feedback
